.gitmodules

```
@@ -1,3 +1,3 @@
-[submodule "concordium-contracts-common"]
-	path = concordium-contracts-common
-	url = ../concordium-contracts-common.git
+[submodule "concordium-rust-sdk"]
+	path = concordium-rust-sdk
+	url = git@github.com:Concordium/concordium-rust-sdk.git
```

This should probably be kept, as a relative path
```suggestion
	url = ../concordium-rust-sdk.git
```

---

Ok, that's fine. I am not sure of the benefits though. Which do you see?

---

It works regardless of whether you check out the repository via https or ssh.

---

Ah, neat.